src/components/ApiReference/ScrollRouter.js

```diff
@@ -5,7 +5,13 @@ import throttle from "lodash/throttle";
 import { smoothScrollTo, findActiveNode } from "helpers/dom";
 import { SideNavProgressContext } from "components/SideNav/Provider";
 
-export const Context = React.createContext();
+export const Context = React.createContext({
+  isScrollingDown: false,
+  stopTrackingElement: () => {},
+  trackElement: () => {},
+  onLinkClick: () => {},
+  setIsNavClicked: () => {},
+});
```

Comment:
I needed to add defaults here because the single API reference pages don't have the provider. The value provided here gets used if the context is consumed without a provider at the top level, so just stubs.